specification/validated-relationships-service-api.yaml

```diff
@@ -279,6 +279,7 @@ paths:
               examples:  
                 internalServerError:
                   $ref: '#/components/examples/InternalServerError'
+
```

Comment:
@chris-young-12-nhs Once this is merged in it will begin breaking the API tests due to the schema validation in the API tests. We should determine the best course of action to ensure we meet schema compliance without harming development

Reply:
@JackPlowman agreed. I suggest before we merge we update the tests on a branch that points to this rev of the spec using the `API_SPEC_BRANCH` env var. Then we can co-ordinate the merge of the spec and the updated tests.